lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -144,6 +168,32 @@ class AwsCompileEventBridgeEvents {
               );
             }
 
+            if (RetryPolicy) {
+              if (!shouldUseCloudFormation) {
+                throw new ServerlessError(
+                  'Configuring RetryPolicy is not supported for EventBrigde integration backed by Custom Resources. Please use `provider.eventBridge.useCloudFormation` setting to use native CloudFormation support for EventBridge.',
+                  'ERROR_INVALID_RETRY_POLICY_TO_EVENT_BUS_CUSTOM_RESOURCE'
+                );
+              }
+              RetryPolicy = _.mapKeys(
+                RetryPolicy,
+                (value, key) => key[0].toLocaleUpperCase() + key.slice(1)
+              );
+            }
+
+            if (DeadLetterConfig) {
+              if (!shouldUseCloudFormation) {
+                throw new ServerlessError(
+                  'Configuring DeadLetterConfig is not supported for EventBrigde integration backed by Custom Resources. Please use `provider.eventBridge.useCloudFormation` setting to use native CloudFormation support for EventBridge.',
+                  'ERROR_INVALID_DEAD_LETTER_CONFIG_TO_EVENT_BUS_CUSTOM_RESOURCE'
+                );
+              }
```

Comment:
@medikoo I'm not sure if you are interested in adding support for `DeadLetterConfig` and `RetryPolicy` using custom resources. As I understand, that is being deprecated, so it's a bit pointless to extend on that. Do you agree with what we made here?

Reply:
@jpbochi yes, exactly, we already agreed to further extend only CF implementation. No point in investing further work into custom resource one